runtime.txt

```diff
@@ -1 +1 @@
-python-3.9
+python-3.10
```

Comment:
Could we bump this more? I mean we should test with older version, but binder could run on latest and greatest.

Reply:
I think we wanted to keep this on the min supported version so all the deps also work fine? I don't exactly remember why we don't use the latest python version, maybe @rossbar ?

Reply:
Yes, but CI can be used to ensure that, I mean there should be a newer combination where all dependencies work.